src/test/ui/impl-trait/auto-trait-leak.stderr

```diff
@@ -1,29 +1,67 @@
-error[E0391]: cycle detected when processing `cycle1`
+error[E0391]: cycle detected when processing `cycle1::{{impl-Trait}}`
```

Comment:
we should think about making this error better, but that seems orthogonal from this PR